cmd/argocd/commands/app.go

```diff
@@ -1521,6 +1582,17 @@ func printAppResources(w io.Writer, app *argoappv1.Application) {
 	}
 }
 
+func printTreeView(nodeMapping map[string]argoappv1.ResourceNode, parentChildMapping map[string][]string, parentNodes map[string]void, mapNodeNameToResourceState map[string]*resourceState) {
+	w := os.Stdout
+	tbl := uitable.New()
+	tbl.Separator = "  "
```

Comment:
Argo CD uses `TAB` as the default separator in the CLI. Why you are changing it to 2 spaces? Is this really required?

Reply:
I see that tab is equivalent to 8 spaces and it means we are giving more space in cases not needed. uitable makes use of "  ", and it also dynamically increases the spacing between columns based on the content widths in the columns.